lib/private/Comments/Manager.php

```diff
@@ -677,18 +677,42 @@ public function getNumberOfUnreadCommentsForObjects(string $objectType, array $o
 	 * @since 21.0.0
 	 */
 	public function getNumberOfCommentsForObjectSinceComment(string $objectType, string $objectId, int $lastRead, string $verb = ''): int {
+		if ($verb !== '') {
+			return $this->getNumberOfCommentsWithVerbsForObjectSinceComment($objectType, $objectId, $lastRead, [$verb]);
+		}
+
 		$query = $this->dbConn->getQueryBuilder();
 		$query->select($query->func()->count('id', 'num_messages'))
 			->from('comments')
 			->where($query->expr()->eq('object_type', $query->createNamedParameter($objectType)))
 			->andWhere($query->expr()->eq('object_id', $query->createNamedParameter($objectId)))
 			->andWhere($query->expr()->gt('id', $query->createNamedParameter($lastRead)));
 
-		if ($verb !== '') {
-			$query->andWhere($query->expr()->eq('verb', $query->createNamedParameter($verb)));
-		}
+		$result = $query->executeQuery();
+		$data = $result->fetch();
+		$result->closeCursor();
 
-		$result = $query->execute();
+		return (int) ($data['num_messages'] ?? 0);
+	}
+
+	/**
+	 * @param string $objectType
+	 * @param string $objectId
+	 * @param int $lastRead
+	 * @param string[] $verbs
+	 * @return int
+	 * @since 24.0.0
+	 */
```

Comment:
```suggestion
	/**
	 * @param string[] $verbs
	 * @since 24.0.0
	 */
```

I would avoid duplicating type information in the docblock for things which are already fully typed in the signature

Reply:
Then phpstorm complains. It's either all or none 🤷‍♂️

Reply:
This is really weird, you cannot configure this?
Rector for instance will remove such statements when using the DEAD_CODE rector.
Anyway it’s not a big deal it’s just visual clutter and duplicated information. For me docblocks should be used when information is missing.